cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractUploaderTest.java

```diff
@@ -268,13 +268,19 @@ public void testImageUploadTag() {
         assertTrue(tag.contains("class='cloudinary-fileupload myclass'"));
     }
 
-
     @Test
     public void testSprite() throws Exception {
         final String sprite_test_tag = String.format("sprite_test_tag_%d", new java.util.Date().getTime());
-        cloudinary.uploader().upload(SRC_TEST_IMAGE, asMap("tags", new String[]{sprite_test_tag, SDK_TEST_TAG, UPLOADER_TAG}, "public_id", "sprite_test_tag_1" + SUFFIX));
-        cloudinary.uploader().upload(SRC_TEST_IMAGE, asMap("tags", new String[]{sprite_test_tag, SDK_TEST_TAG, UPLOADER_TAG}, "public_id", "sprite_test_tag_2" + SUFFIX));
-        Map result = cloudinary.uploader().generateSprite(sprite_test_tag, asMap("tags", Arrays.asList(SDK_TEST_TAG, UPLOADER_TAG)));
+        Map uploadResult1 = cloudinary.uploader().upload(SRC_TEST_IMAGE, asMap("tags", new String[]{sprite_test_tag, SDK_TEST_TAG, UPLOADER_TAG}, "public_id", "sprite_test_tag_1" + SUFFIX));
+        Map uploadResult2 = cloudinary.uploader().upload(SRC_TEST_IMAGE, asMap("tags", new String[]{sprite_test_tag, SDK_TEST_TAG, UPLOADER_TAG}, "public_id", "sprite_test_tag_2" + SUFFIX));
+
+        String[] urls = new String[]{uploadResult1.get("url").toString(), uploadResult2.get("url").toString()};
+
+        Map result = cloudinary.uploader().generateSprite(urls, asMap("tags", Arrays.asList(SDK_TEST_TAG, UPLOADER_TAG)));
```

Comment:
Why do we need the `tags` option?

Reply:
`tags` here just add these tag to the generated sprite (it's not the tag to filter by). We tag anything we can in the tests to better track issues.